packages/storybook-html/src/components/breadcrumb/breadcrumb.docs.mdx

```diff
@@ -2,7 +2,7 @@ import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
 import StatusBadge from '../../../.storybook/custom/components/StatusBadge';
 
 # Breadcrumb
-<StatusBadge.Stable />
+<StatusBadge.Review />
```

Comment:
I will be removing the "review" flag. Leave as "stable" as this is technically a bug fix and no breaking changes should occur as a result.

Reply:
Done